plugins/schedule/base/lib/api/KalturaScheduleEvent.php

```diff
@@ -352,6 +406,25 @@ public function validateForUpdate($sourceObject, $propertiesToSkip = array())
 			$this->id = $sourceObject->getId();
 		}
 		
+		$linkedTo = $sourceObject->getLinkedTo();
+		if ($this->linkedTo && $this->linkedTo->getEventId() && $this->startDate)
+		{
+			throw new KalturaAPIException(KalturaScheduleErrors::START_TIME_AND_LINKED_TO_CONFLICT);
+		}
+		elseif ($this->linkedTo)
+		{
+			$this->setTimingFromLinkedToEvent($this->linkedTo->getEventId(), $sourceObject->getDuration());
+			if ($this->startDate && $this->endDate)
+			{
+				$startDate = $this->startDate;
+				$endDate = $this->endDate;
+			}
+		}
+		if ((!is_null($this->linkedTo) || $sourceObject->getLinkedTo()) && $this->recurrenceType != KalturaScheduleEventRecurrenceType::NONE)

```

Comment:
Again, why is double validation needed here? Plus it comes after aready handling linkedTo

Reply:
here we are validating a different scenario, when the user is trying to change the event into a recurring event while also having a linkedTo property set. As we discussed, this cannot be allowed, and we throw an exceptiom.